### ISSUE TITLE
Narrator doesn't read expanded information on Dropdown

### ISSUE THREAD

**Issue by cnamal** [OfficeDev/office-ui-fabric-react#9659]
Go to https://developer.microsoft.com/en-us/fabric#/controls/web/dropdown
Start Narrator
Tab until you get to the _Basic uncontrolled example_ dropdown.

Narrator doesn't convey the `aria-expanded` information. 

I saw this [comment](https://github.com/OfficeDev/office-ui-fabric-react/issues/6326#issuecomment-493503894), however the linked W3 documentation doesn't use `aria-expanded`, so it's incomparable IMO. Might still be a Narrator issue though :)

**Comment by khmakoto** [comment]
Hi @cnamal! Thanks for filing the issue! I can confirm that this is indeed happening. I'll take a look to see if this is our issue or an issue with Narrator. Thanks!

**Comment by cnamal** [comment]
the same is true for Combobox.

**Comment by cnamal** [comment]
W3C provides a working Combobox:
https://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.1pattern/listbox-combo.html
I think the difference is W3C's expanded information is on a div, and on OUFR it's on an input

**Comment by eveleens** [comment]
Any traction on this one?

**Comment by khmakoto** [comment]
Hi @eveleens still waiting for a response on this one. The narrator team has not been very responsive lately.

**Comment by kenotron** [comment]
I've linked the internal Narrator team via their Feedback Hub on Windows with this link:
https://www.w3.org/TR/wai-aria-practices/examples/listbox/listbox-collapsible.html

Which is closer in functionality than a ComboBox (simpler). We cannot action on this since we're already applying the correct aria-expanded state already. I'm closing this one - it is an external issue.
